msr206.c

```diff
@@ -5,6 +5,10 @@
 #include <strings.h>
 #include <err.h>
 #include <string.h>
+#include <math.h>
```

Comment:
Could you explain this changeset? The codebase currently builds for me without these includes, and raw writes shouldn't require a larger buffer.